open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -1755,18 +1757,9 @@ PROCEDURE SHAREOUTPUT
        IF SACSHARETYPEEXISTS(J)=TRUE AND
```

Comment:
Hi @rrobison44, I agree what you did looks better. Code changes look good. Approved.
My one recommendation is to add a NEWLINE right above FOR J=0 TO SACSHARETYPESMAX (the loop you are addressing) so that first new Share Type '{' (open bracket) lines up well (only focused on that as it seems the Print statements are using spaces before '{' trying to make it line up.  Thank you.